src/api/types.ts

```diff
@@ -21,13 +21,13 @@ interface ScorecardCreator {
 
 export interface Scorecard {
   creator: ScorecardCreator;
+  description?: string;
+  filter?: EntityFilter | null;
```

Comment:
In what scenario does the `filter` property not exist on a Scorecard?

Reply:
I don't know anything about such scenario, however I had to add `?` as in `hooks.ts` is `useScorecardCompareFn` defined, which uses `customCompareFn` (which may be defined by user (?)) which (for some reason) calls `nullsToUndefined` on each scorecard first.

It was added there for some reason, so I didn't want to change that, however I had to extend Scorecard type definition.

Reply:
Ugh alright